jbi/steps.py

```diff
@@ -28,22 +32,30 @@ def create_comment(context: ActionContext, parameters: ActionParams):
         )
         return context
 
-    jira_response = jira.add_jira_comment(context)
+    jira_response = jira_service.add_jira_comment(context)
     context = context.append_responses(jira_response)
     return context
 
 
-def create_issue(context: ActionContext, parameters: ActionParams):
+def create_issue(
+    context: ActionContext,
+    *,
+    parameters: ActionParams,
+    jira_service: JiraService,
+    bugzilla_service: BugzillaService,
+):
     """Create the Jira issue with the first comment as the description."""
     bug = context.bug
     issue_type = parameters.issue_type_map.get(bug.type or "", "Task")
 
     # In the payload of a bug creation, the `comment` field is `null`.
     # We fetch the list of comments to use the first one as the Jira issue description.
-    comment_list = bugzilla.get_client().get_comments(bug.id)
+    comment_list = bugzilla_service.client.get_comments(bug.id)
```

Comment:
Yeah, this is a good idea. I made some changes to avoid direct client access here [653b4a4](https://github.com/mozilla/jira-bugzilla-integration/pull/621/commits/653b4a4c6afc5b7262097cb72b7ed35be7da158e)

Probably worth another review.